realm/realm-library/src/objectServer/java/io/realm/UserStore.java

```diff
@@ -43,22 +43,28 @@
 
     /**
      * Retrieves the current {@link SyncUser}.
+     * <p>
+     * This method will throw an exception if more than one valid, logged in users exist.
+     * @return {@link SyncUser} object or {@code null} if not found.
+     */
+    SyncUser getCurrent();
```

Comment:
I don't think that we should have `removeCurrent()` since it is easy to implement by using `getCurrent()` and `remove(String)`.